vector/src/main/java/im/vector/app/features/widgets/WidgetAction.kt

```diff
@@ -26,4 +26,5 @@ sealed class WidgetAction : VectorViewModelAction {
     object DeleteWidget : WidgetAction()
     object RevokeWidget : WidgetAction()
     object OnTermsReviewed : WidgetAction()
+    object HangupElementCall : WidgetAction()
```

Comment:
Could we rename the action `CloseWidget` to avoid introducing too specific action?

Reply:
Renamed.